MAUI/PDF-Viewer/Free-Text.md

```diff
@@ -184,11 +196,11 @@ Private void PdfViewer_FreeTextModalViewDisappearing(object? Sender, EventArgs e
 {% endhighlight %} 
 {% endtabs %}
 
-### Supressing the free text modal view and implement your own UI
+### Suppressing the free text modal view and implement your own UI
 
-The [Sfpdfviewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to supress the free text modal view and use your own UI in its place. This can be achieved by setting the `AnnotationModalViewAppearingEventArgs.Cancel` property to `true` in the `FreeTextModalViewAppearing` event handler. 
+The [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to suppress the free text modal view and use your own UI in its place. This can be achieved by setting the `AnnotationModalViewAppearingEventArgs.Cancel` property to `true` in the `FreeTextModalViewAppearing` event handler. 
```

Comment:
Since the APIs are in live now, can this be API reference linked?

Reply:
Feedback addressed